pkg/specgen/generate/kube/seccomp.go

```diff
@@ -11,6 +11,7 @@ import (
 
 // KubeSeccompPaths holds information about a pod YAML's seccomp configuration
 // it holds both container and pod seccomp paths
+// nolint:golint
```

Comment:
What was the linter complaining about here and at the others structs?

Reply:
```
pkg/specgen/generate/kube/seccomp.go:14:6: type name will be used as kube.KubeSeccompPaths by other packages, and that stutters; consider calling this SeccompPaths (golint)
type KubeSeccompPaths struct {
```
However the variable `SeccompPaths` is already used in this package so I just added the nolint

Reply:
Ah, thanks!